core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BpmBillingProjectLifecycle.scala

```diff
@@ -214,7 +216,10 @@ class BpmBillingProjectLifecycle(
   private def cleanupLandingZone(
     landingZoneId: UUID,
     ctx: RawlsRequestContext
-  ): DeleteAzureLandingZoneResult = Try(workspaceManagerDAO.deleteLandingZone(landingZoneId, ctx)) match {
+  ): Option[DeleteAzureLandingZoneResult] = Try(workspaceManagerDAO.deleteLandingZone(landingZoneId, ctx)) match {
```

Comment:
Made the return an optional value, so we could indicate a non-failure instance where we don't need to create the monitor job to wait for the result. 
This matches up nicely with the result and return behavior of the initializeDelete method, and doesn't conflict with the usage in cleanup when creation fails.